kubernetes-ingress.md

```diff
@@ -0,0 +1,150 @@
+# Ingress
+[⬅️ Back to Kubernetes overview](README.md)
+
+
+## Setup ingress controller
+![Ingress Controller](./kubernetes/ingress.png)
```

Comment:
I don't think this graph helps at this point in the documentation - we haven't talked about nginx yet, and I guess many people don't recognize the nginx logo. Maybe a short subtext would help: *This overview shows a nginx ingress controller, containing a self-managed nginx deployment. The controller manages ingress resources from e.g. your service.*